packages/flutter_tools/test/general.shard/base/logger_test.dart

```diff
@@ -936,51 +932,6 @@ void main() {
       expect(lines[0], equals('All good.'));
     });
 
-    testWithoutContext('Stdout printStatus handle null inputs on colored terminal', () async {
```

Comment:
Is this no longer a legitimate test?

Reply:
You can't pass null to Logger.printStatus anymore.

Reply:
SGTM